cmd/collectors/zapi/plugins/quota/quota.go

```diff
@@ -150,7 +162,7 @@ func (my *Quota) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 			return nil, errors.New(errors.ERR_NO_INSTANCE, "no quota instances found")
 		}
 
-		my.Logger.Debug().Msgf("fetching %d quota counters", len(quotas))
+		my.Logger.Debug().Msgf("fetching %d quotas", len(quotas))
```

Comment:
minor but since we're touching this log message can you go ahead and switch to the typed version with `Msg`? 
`my.Logger.Debug().Int("quotas", quotas).Msg("Fetching quotas")`

Reply:
Done